trigger/commit_queue.go

```diff
@@ -92,14 +94,19 @@ func (t *commitQueueTriggers) makeData(sub *event.Subscription) (*commonTemplate
 	if t.patch.Version != "" {
 		url = fmt.Sprintf("%s/version/%s", t.uiConfig.Url, t.patch.Version)
 	}
+	projectName := t.patch.Project
+	identifier, err := model.GetIdentifierForProject(t.patch.Project)
+	if err != nil && identifier != "" {
+		projectName = identifier
```

Comment:
Should we be returning an error in this case?

Reply:
I think it's better to still send the notification, since we can just use what's in t.patch.Project in the case of a database error.

Reply:
Oh whoops but I should be saying if err == nil, not if err != nil

Reply:
Doesn't an error here indicate that something went wrong and we should do something about it?

Reply:
that makes sense, thanks!